docs(instana-aws-lambda-auto-wrap): added note about deprecating npm package (#1372)
instana/nodejs: packages/aws-lambda-auto-wrap/README.md

This mainly exists because AWS Lambda handlers cannot be scoped packages.

## Deprecation Notice

> **Important:** This is an internal package will soon be deprecated.  

Reviewer: ```suggestion
**Important:** This is an internal and private package that will soon be deprecated.
```

Reviewer: > This is an internal and private package that will soon be deprecated.

This is an internal and private package that will soon be **removed**.

> - We will no longer publish new releases to NPM.

Reviewer: ```suggestion
- We are no longer publish the package to npm.
```
